api/serializers.py

```diff
@@ -108,3 +108,24 @@ def to_representation(self, instance):
         representation['object'] = representation['post']['source']
         del representation['post']
         return representation
+
+
+class CommentLikeSerializer(serializers.ModelSerializer):
+    parent_lookup_kwargs = {
+        'author_pk': 'author__pk',
+        'post_pk': 'post__pk',
+    }
+    author = AuthorSerializer(many=False, read_only=True)
+    comment = CommentSerializer(many=False, read_only=True)
+
+    class Meta:
+        model = CommentLike
+        fields = ['author', 'comment']
+
+    def to_representation(self, instance):
+        representation = super().to_representation(instance)
+        representation['type'] = 'Like'
+        representation['summary'] = instance.author.get_full_name() + ' likes your comment'
+        representation['object'] = representation['comment']
```

Comment:
Trying to figure out how to get the source url here

Reply:
```suggestion
        representation['object'] = representation['comment']['url']
```
atm but needs to be updated to `['comment']['id']` to satisfy specs in a follow up PR